WebAssembly.md

```diff
@@ -1764,13 +1764,15 @@ The `call` instruction performs a [call](#calling) to the function with index
 
 #### Indirect Call
 
-| Mnemonic        | Opcode | Immediates                | Signature                                                | Families |
-| --------------- | ------ | ------------------------- | -------------------------------------------------------- | -------- |
-| `call_indirect` | 0x11   | `$signature`: [varuint32] | `($T[`[`$args`]`], $callee: i32) : ($T[`[`$returns`]`])` | [L]      |
+| Mnemonic        | Opcode | Immediates                                         | Signature                                                | Families |
+| --------------- | ------ | -------------------------------------------------- | -------------------------------------------------------- | -------- |
+| `call_indirect` | 0x11   | `$signature`: [varuint32], `$reserved`: [varuint1] | `($T[`[`$args`]`], $callee: i32) : ($T[`[`$returns`]`])` | [L]      |
 
 The `call_indirect` instruction performs a [call](#calling) to the function in
 the default table with index `$callee`.
 
+In future versions of WebAssembly, the reserved immediate may be used to index additional tables.
```

Comment:
For consistency with other such notes, please prefix this sentence with "> " so that it's formatted as a blockquote, and move it below the `Validation:` clause.